src/components/helpers/css.ts

```diff
@@ -118,14 +118,18 @@ export function cellTextStyleToCss(style: Style | undefined): CSSProperties {
   if (style.textColor) {
     attributes["color"] = style.textColor;
   }
+  if (style.fillColor) {
+    attributes["background"] = style.fillColor;
+  }
 
   return attributes;
 }
 
-export function cssPropertiesToCss(attributes: CSSProperties): string {
+export function cssPropertiesToCss(attributes: CSSProperties, newLine = true): string {
+  const separator = newLine ? "\n" : "";
   const str = Object.entries(attributes)
     .map(([attName, attValue]) => `${attName}: ${attValue};`)
-    .join("\n");
+    .join(separator);
 
   return "\n" + str + "\n";
```

Comment:
maybe don't add both "\n" if str is empty. It will result in a better formatted html string